src/wavelet/mod.rs

```diff
@@ -2053,6 +2053,14 @@ impl WaveletMatrix {
     }
 }
 
+impl From<WaveletMatrix> for BitVec {
```

Comment:
I don't understand the significance of this. A wavelet matrix encodes symbols bit per bit, the n'th bit of each symbol in the n'th RsVec of the WaveletMatrix. Why would we want to append all those vectors into a single vector?

Reply:
It's not something I've needed in my project, I added mostly for symmetry with BpTree. Since both are constructed from a BitVec, I figured it made sense to implement the backwards conversion for both types. If that's nonsense for WaveletMatrix we can delete it.

Reply:
Yeah it doesn't make sense here. The wavelet matrix is constructed from integers, or for very strange inputs from BitVecs, but they are transposed during construction, so appending them like this doesn't make sense.